pages/common/if.md

```diff
@@ -0,0 +1,11 @@
+# if
+
+> Simple shell conditional
+
+- Perform a command depending on another command's success.
+
+`{{cmd}} && echo "success" || echo "failure"``
```

Comment:
This is not if.

BTW, check the [page for test](https://github.com/tldr-pages/tldr/blob/master/pages/common/test.md).